<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>goodsmovement</class>
 <widget class="QWidget" name="goodsmovement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Հաշվ․ համար</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EQLineEdit" name="leInvDoc">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Հաշվարկային ժամանակաշրջան</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EDateEdit" name="date1"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EDateEdit" name="date2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Պահեստ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EQLineEdit" name="leStore"/>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>161</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text">
         <string>Բեռնել</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="EQTableWidget" name="tbl">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>Ապրանքի կոդ</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ապրանքի անվանում</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Նախնական</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Մուտք</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Ելք</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Վերջնական</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
